spec/swagger_helper.rb

```diff
@@ -126,6 +141,46 @@
               qris_rating: { type: :string, example: '1' }
             }
           },
+          case_cycle: {
+            type: :object,
+            properties: {
+              case_number: {
+                anyOf: [
+                  { type: :string, example: '123-45' },
+                  { type: :null }
+                ]
+              },
+              copay_cents: { type: :integer, minimum: 0 },
+              copay_currency: { type: :string, example: 'USD' },
+              copay_frequency: {
+                anyOf: [
+                  { '$ref': '#/components/schemas/copay_frequency' },
+                  { type: :null }
+                ]
+              },
+              effective_on: {
+                anyOf: [
+                  { type: :string, example: '2019-06-27' },
+                  { type: :null }
+                ]
+              },
+              expires_on: {
+                anyOf: [
+                  { type: :string, example: '2020-06-28' },
+                  { type: :null }
+                ]
+              },
+              notified_on: {
+                anyOf: [
+                  { type: :string, example: '2019-06-30' },
+                  { type: :null }
+                ]
+              },
+              status: { '$ref': '#/components/schemas/case_status' },
```

Comment:
```suggestion
              status: { '$ref': '#/components/schemas/case_statuses' },
```

Reply:
👍   I need to update the name in the schema, too.
Good catch.